### ISSUE TITLE
Document compatibility with popular `redux` libraries.

### ISSUE THREAD

**Issue by wtgtybhertgeghgtwtg** [procore/redux-modules#127]
i.e. `redux-form`, `redux-saga`.

**Comment by mboperator** [comment]
I agree with this. Examples with each library would go a long way. Here's a list of which libraries I think we should potentially hit first:

- [ ] redux-thunk
- [ ] redux-loop
- [ ] redux-saga
- [ ] redux-form
